src/stories/StepFunctionGraph/stepFunctionGraphData.ts

```diff
@@ -0,0 +1,573 @@
+export const simpleJson = {
```

Comment:
Check this file for anything that needs to be scrubbed that I may have missed.

Reply:
Data payload looks generic enough.